applications/default/public/js/controllers.js

```diff
@@ -304,6 +304,9 @@ function ViewController($scope, $location, $http, applicationState, Choko) {
 
     Choko.get({type: $scope.view.itemType}, function(response) {
       $scope.items = response;
+      $scope.items.$empty = Object.keys($scope.items).filter(function (key) {
+        return key.indexOf('$') != 0;
```

Comment:
Maybe you could use $scope.items.hasOwnProperty(key) for checking if the property is local or inherited.